### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by copy-frogger** [boyuzz/WMCNN-Pytorch#3]
Hello, I would like to ask when training loss is always more than 120, is this normal?

**Comment by boyuzz** [comment]
Sorry for late reply. The loss is not averaged over the batch so more than 120 is normal.
